src/openMVG/features/svgVisualization.cpp

```diff
@@ -24,11 +51,11 @@ float getStrokeEstimate(const std::pair<size_t,size_t> & imgSize)
 
 void saveMatches2SVG(const std::string &imagePathLeft,
                      const std::pair<size_t,size_t> & imageSizeLeft,
-                     const std::vector<features::PointFeature> &keypointsLeft,
+                     const features::MapRegionsPerDesc &keypointsLeft,
```

Comment:
i'd prefer to have a low level function that just draw a single keypoint type, like it was before.
We can add another function with this signature that call that function  (we can add the color param to the original one)